tower-buffer/src/service.rs

```diff
@@ -93,7 +93,9 @@ where
         // outside of task context.
         let (tx, rx) = oneshot::channel();
 
-        match self.tx.try_send(Message { request, tx }) {
+        let span = tracing::Span::current();
```

Comment:
TIOLI: Might be nice to have a comment here for future readers noting that we capture the current span here to propagate it to the buffer worker?